### DISCUSSION TITLE
Upgrade existing tea-cli and "replace" references

### DISCUSSION THREAD

**Discussion by randy** [pkgxdev/pkgx#870] [Help] [answered]
Hi!

I'm just now catching up no the rename, etc. I've tried searching for any other "help" discussions regarding this, but not seeing anything; if I've missed it, please link me there!

So, I've had the `tea` CLI installed on my older MBP for awhile, but hadn't used it beyond having a line in my `.zshrc` file that I'm not even sure about now: `add-zsh-hook -Uz chpwd(){ source <(tea -Eds) }`.

What I'm looking for help on is, first, what do I do on my old laptop to update to the latest version of the product? `brew` doesn't know anything about `tea-cli` and I honestly don't remember how I installed it in the first place... Maybe I just let it be and install `pkgx`?

Second, I've copied things like my `.zshrc` file to my new laptop, so that line above is causing the shell to issue warnings. Yes, I can just drop the line, but being that I haven't used `tea` and I'm not entirely sure what all it installed, I'm not sure what other leftovers are lurking. For what's it's worth, my new laptop is also a MBP and I didn't use Migration Assistant, so I've got a fairly clean slate here. For now, I'm just commenting that line out, but I'd like to know if there might be any other cobwebs around that need clearing.

Thanks!

**Comment by mxcl** [comment, marked as answer]
Hey! So yeah, we have a migration guide, but I removed it from the README at 1.0:

https://blog.pkgx.dev/pkgx-1-0-0-alpha-1/

But we can help you directly here. Remove that line from your zshrc, then `which tea` and delete it. Then install pkgx and `pkgx integrate` and you should be fine.

**Reply by randy** [reply]
Looks like I'd already removed it, so I just cleared out the comment line and did the install per the README. No errors, so assuming that worked! I'll reply here if there are any troubles when I get to using it. Thanks!
